### PR TITLE
Hub: Add hub token copy feature

### PR DESCRIPTION
**Pull request by Divyansh42**
# Changes

Add feature to copy hub token from the frontend and also modified UI in the toolbar.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: hub/ui/public/config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 window.config = {
   API_URL: 'https://api.hub.tekton.dev',
-  GH_CLIENT_ID: '79b882442755f8973034',
+  GH_CLIENT_ID: '989e7cb9be3bee8e3c47',
```
Comment (PuneetPunamiya) [inline hub/ui/public/config.js:3]: Do we need this change ??

Comment (Divyansh42) [reply hub/ui/public/config.js:3]: Nope, will remove this.

File: hub/ui/src/components/main/App.tsx [hunk 2]
```diff
@@ -55,23 +61,23 @@ const App: React.FC<mainProps> = () => {
...
         }}>
         Login</span>
     </Link>;
     displayUpload = '';
   } else {
-    authenticationButton = <Link to="/">
-      <span style={{marginRight: '1em', color: 'white', fontSize: '1em'}}
-        onClick={logoutUser}> Logout </span>
-    </Link>;
+    // authenticationButton = <Link to="/">
```
Comment (PuneetPunamiya) [inline hub/ui/src/components/main/App.tsx:77]: Can we remove this ??

**Review by PuneetPunamiya — COMMENTED** [review]
@Divyansh42  Thank you for the patch, the code looks fine to me :+1:

File: hub/ui/src/components/main/App.tsx [hunk 3]
```diff
@@ -108,23 +117,57 @@ const App: React.FC<mainProps> = () => {
 
           <ToolbarItem>
             {
-              authenticationButton
+              loginButton
```
Comment (sthaha) [inline hub/ui/src/components/main/App.tsx:120]: wouldn't this be `undefined` if there is no token?

**Pull request merged**